src/main/main.ts

```diff
@@ -291,10 +300,19 @@ ipcMain.on('settingsWindow', (event, args) => {
   }
     
   if (settingsWindow === null) return; 
-
+  
   if (args[0] === "quit") {
     console.log("User closed settings");
     settingsWindow.close();
+    storageDir = getPathConfigSafe(cfg, 'storage-path');
+    baseLogPath = getPathConfigSafe(cfg, 'log-path');
+    maxStorage = getNumberConfigSafe(cfg, 'max-storage');
+    if (checkConfig()) {
```

Comment:
I'm a stickler for a new line before a condition like this x)

Reply:
agree on this. makes the code easier to read.